### DISCUSSION TITLE
Setting default window views? And running macros.

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#374] [🙋♀️TE3 Q&A] [answered]
Is there a way to change the default window sizes in TE/3? With the 3.2 update, the right pane is narrower, so long names in the TOM or rules in the BPA are truncated a bit. I want to make that wider for my session, but am not seeing a way to save that view. Am I missing something?

I also recently discovered I can right-click on an object in the TOM and run macros that are scoped to that level. So right-clicking on Model allows me to format dax. However, it does so invisibly. If I go to the Macros tab, click on the macro I want, then run, I get a window that says "executing script" and it does the work.

But if I right-click on model, Macros, Format Dax, it does run, but invisibly, so that feedback is missing. I had to verify it was actually working. Is that expected behavior?

**Comment by otykier** [comment, marked as answer]
(Please feel free to create separate issues for separate questions/bugs).

TE3 should save the width of the pane when you exit the application or when you use the "Capture layout" option under the **Windows** menu. Upon application re-launch, the width should be restored.

Regarding your other point, I agree - we should have some kind of visual indication that a macro has executed (for example, by displaying in the status bar the number of changes applied, similar to when a script is executed).

**Comment by edhans** [comment]
Thanks. Wow, a whole new menu of layouts to save and switch between! Thanks!
